app/views/teacher_interface/application_forms/show/_declined.html.erb

```diff
@@ -9,17 +9,27 @@
     <h4 class="govuk-heading-s"><%= title %></h4>
   <% end %>
 
-  <% if reasons.count == 1 %>
-    <%= simple_format reasons.first %>
+  <% if reasons.size == 1 %>
+    <%= simple_format reasons.first[:decline_reason] %>
+    <% if reasons.first[:assessor_note].present? %>
+      <p><%= simple_format reasons.first[:assessor_note] %></p>
```

Comment:
```suggestion
      <%= simple_format reasons.first[:assessor_note] %>
```

`simple_format` already returns a paragraph tag so you don't need to wrap this in a paragraph.